Microsoft.ContentModerator.AMSComponent/AMSComponentClient/VideoReviewAPI.cs

```diff
@@ -380,7 +381,10 @@ private VideoFrame PopulateVideoFrame(ProcessedFrameDetails frameEvent, UploadAs
                         new Metadata() {Key = "a", Value = frameEvent.IsAdultContent.ToString()},
                         new Metadata() {Key = "Racy Score", Value = frameEvent.RacyScore},
                         new Metadata() {Key = "r", Value = frameEvent.IsRacyContent.ToString()},
-                        new Metadata() {Key = "ExternalId", Value = frameEvent.FrameName}
+                        new Metadata() {Key = "ExternalId", Value = frameEvent.FrameName},
```

Comment:
Why use the short forms in metadata ?  It will be hard to understand what "at", "rt" means. 

I suggest that we change these to AdultText, RacyText, OffensiveText